lib/config.js

```diff
@@ -35,6 +36,7 @@ var path = require('path'),
 var config = {
     configFile: path.join(__dirname, '/../config', name + '.yml'),
     listenPort: 3000,
```

Comment:
if config/aiakos.yml:app:port is 3000 why are you puttinh here the value and not the config data?